src/css/tooltip.less

```diff
@@ -1,53 +0,0 @@
-/**
- * Tooltip
- */
-
-@-webkit-keyframes fade-in {
-  0% { opacity: 0; }
-  100% { opacity: 1; }
-}
-@keyframes fade-in {
-  0% { opacity: 0; }
-  100% { opacity: 1; }
-}
-
-.__mobiledoc-tooltip {
```

Comment:
Tooltips are still used, actually. The functionality should be extracted to the Ember editor before we drop it here. It will ensure we have appropriate APIs exposed.

![](https://api.monosnap.com/rpc/file/download?id=oVCbPumkPVfDcok05AmIsQyWPKh6Gm)